Core/src/main/java/properties/FileState.java

```diff
@@ -0,0 +1,5 @@
+package properties;
+
+public enum FileState {
```

Comment:
@michalryn While I was browsing the db created by Drifty just now, I saw it stores `State` as numbers. Just curious, what do each numbers mean? Are they the enum values?
![image](https://github.com/user-attachments/assets/69290d85-c523-4a4e-9ee7-f4e938f4bbc8)